test/stop.t

```diff
@@ -198,6 +198,18 @@ class TestStop(TestCase):
         self.assertEqual(len(j), 1)
         self.assertClosedInterval(j[0])
 
+    def test_stop_with_id(self):
+        """Stop does not work with with ids other than one"""
+        self.t("start 2h ago")
+        self.t("start 1h ago")
+        
+        self.t("stop @1")
+
+        code, out, err = self.t.runError("stop @2")
+        # If trying to stop an older interval, check that modify was
+        # suggested.
```

Comment:
There is no 80-char-linelength-rule, use the space. If, then it is 'one sentence, one line' :)

Reply:
Done in sruffell/timewarrior@c9325f1